app/liquid_tags/stackblitz_tag.rb

```diff
@@ -4,7 +4,7 @@ class StackblitzTag < LiquidTagBase
   ID_REGEXP = /\A(?<id>[\w-]{,60})\Z/
   REGEXP_OPTIONS = [REGISTRY_REGEXP, ID_REGEXP].freeze
   # rubocop:disable Layout/LineLength
-  PARAM_REGEXP = /\A(view=(preview|editor|both))|(file=(.*))|(embed=1)|(hideExplorer=1)|(hideNavigation=1)|(theme=(default|light|dark))|(ctl=1)|(devtoolsheight=\d)\Z/
+  PARAM_REGEXP = /\A(view=(preview|editor|both))|(file=(.*))|(embed=1)|(hideExplorer=1)|(hideNavigation=1)|(theme=(default|light|dark))|(ctl=1)|(devtoolsheight=\d)|(hidedevtools=1)|(initialpath=(.*))|(showSidebar=1)|(terminalHeight=\d)|(startScript=(.*))\Z/
```

Comment:
"showSidebar" seems to have no effect, as according to the [docs](https://developer.stackblitz.com/guides/integration/embedding#embed-url-options), Forem's article width does not have enough space to display the sidebar.

> Shows the sidebar in embed view (large viewports only)